From 6171583033ffa13497fc8ccd6fed63aacccbce48 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 24 Sep 2018 10:59:52 +0300
Subject: [PATCH] Goya UART CLK: update according to new PLL div take2

In order to support ARM timestamps by default.
changed div to PCI[div1] to be 35.
It means that UART CLK is 860M/(35+1) =~ 23.88MHz
---

diff --git a/board/habanalabs/goya/uboot_asic.fragment b/board/habanalabs/goya/uboot_asic.fragment
index c9d383c..83f2fc2 100644
--- a/board/habanalabs/goya/uboot_asic.fragment
+++ b/board/habanalabs/goya/uboot_asic.fragment
@@ -2,7 +2,7 @@
 CONFIG_CONARGS="serial0:115200n8"
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
-CONFIG_DEBUG_UART_CLOCK=50588000
+CONFIG_DEBUG_UART_CLOCK=23888888
 CONFIG_BOOTCOMMAND="getfit && bootm $fileaddr#conf@1"
 CONFIG_DEFAULT_DEVICE_TREE="goya_asic"
 CONFIG_SPI_FLASH_MACRONIX=y
